app/controllers/users_controller.rb

```diff
@@ -37,6 +37,6 @@ def show
   end
 
   def user_params
-    params.require(:user).permit(:email, :password, :full_name)
+    params.require(:user).permit(:email, :password, :full_name, :customer_token)
```

Comment:
Is `customer_token` something that User has to provide/type himself to form? If not then it should not be in strong params.